runtests.py

```diff
@@ -4,13 +4,13 @@
 import os
 import sys
 
-sys.path.insert(0, "tests")
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+sys.path.insert(0, 'tests')
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'openwisp2.settings')
 
-if __name__ == "__main__":
+if __name__ == '__main__':
     from django.core.management import execute_from_command_line
 
     args = sys.argv
-    args.insert(1, "test")
-    args.insert(2, "openwisp_notifications")
+    args.insert(1, 'test')
+    args.insert(2, 'openwisp_notifications')
```

Comment:
I think the reason is here. Look at the runtests.py script in the modules which implement swappable (eg: firmware-upgrader)